numpy/core/tests/test_indexing.py

```diff
@@ -1171,8 +1167,9 @@ def test_bool_as_int_argument(self):
         # array is thus also deprecated, but not with the same message:
         assert_raises(TypeError, operator.index, np.array(True))
         assert_raises(TypeError, np.take, args=(a, [0], False))
-        assert_raises(IndexError, lambda: a[False, 0])
-        assert_raises(IndexError, lambda: a[False, 0, 0])
+        #fixme
```

Comment:
The whole test should probably be replaced with new tests for the new behaviour. Sounds like it tests old behaviour